src/swarmus_ros_navigation/rviz/agent_1_config.rviz

```diff
@@ -213,9 +206,9 @@ Visualization Manager:
       Enabled: true
       Invert Rainbow: false
       Max Color: 255; 255; 255
-      Max Intensity: 0
+      Max Intensity: -999999
       Min Color: 0; 0; 0
-      Min Intensity: 0
+      Min Intensity: 999999
```

Comment:
Was the min and max intensity set just to test or those are real values?

Reply:
I don't know how it was set to those values but it shouldn't. Even launching this Rviz config brings those values to 0 so they dont even have an impact. I commited a fix to avoid further confusion.